test/AutoRest.TestServerLowLevel.Tests/LowLevel/Generation/ReadonlyPropertyWritingTests.cs

```diff
@@ -151,6 +163,8 @@ public partial class RoundTripModel: global::Azure.Core.IUtf8JsonSerializable
 int requiredReadonlyInt = default;
 global::Azure.Core.Optional<string> optionalReadonlyString = default;
 global::Azure.Core.Optional<int?> optionalReadonlyInt = default;
+global::Cadl.TestServer.ReadonlyProperties.Models.ReadonlyModel requiredReadonlyModel = default;
```

Comment:
Are those generated codes? why do they use the type path, not use `using` to import?

Reply:
This is not the final codes. It's the intermediate form which will be processed by Roslyn and then generate the final codes.